src/components/button/bl-button.css

```diff
@@ -68,6 +68,19 @@
   outline: none;
 }
 
+.button:focus-visible {
+  position: relative;
+  outline: none;
```

Comment:
When I see this, I just asked myself "why we are not just using outline and adding so many stuff like this" and realized that [there is a bug](https://bugs.webkit.org/show_bug.cgi?id=246192) in webkit about using `outline` with `border-radius`. It's actually fixed and release in [Safari Techonology Preview 157](https://webkit.org/blog/13575/release-notes-for-safari-technology-preview-157/) last year, but it's somehow not released yet in stable release. It's sad that this bug adds extra complication here.